index.js

```diff
@@ -82,6 +101,14 @@ const preprocessDestinationPath = ({entry, destination, options}) => {
 		return path.join(destination, entry.name);
 	}
 
+	if (entry.pattern.isDirectory && path.relative(options.cwd, entry.path).startsWith('..')) {
```

Comment:
This check will not work correctly if `options.cwd` and `entry.path` are on different partitions on Windows, see: https://github.com/sindresorhus/import-local/pull/12